src/utils/__tests__/findListStyleType-test.js

```diff
@@ -0,0 +1,139 @@
+import test from "tape"
+import findListStyleType from "../findListStyleType"
+
+test("findFontFamily", t => {
+  t.deepEqual(
+    findListStyleType("inherit"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "inherit",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("none"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "none",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("circle"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "circle",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("inside"),
+    []
+  )
+  t.deepEqual(
+    findListStyleType("url('kayo.jpg')"),
+    []
+  )
+  t.deepEqual(
+    findListStyleType("url('marker.gif') inside"),
+    []
+  )
+  t.deepEqual(
+    findListStyleType("square outside"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "square",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("inside square"),
+    [{
+      sourceIndex: 7,
+      type: "word",
+      value: "square",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("square inside url('sqpurple.gif')"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "square",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("url('sqpurple.gif') square inside"),
+    [{
+      sourceIndex: 20,
+      type: "word",
+      value: "square",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("square url('sqpurple.gif') inside"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "square",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("custom-counter-style"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "custom-counter-style",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("customCounterStyle"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "customCounterStyle",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("inside custom-counter-style"),
+    [{
+      sourceIndex: 7,
+      type: "word",
+      value: "custom-counter-style",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("custom-counter-style outside"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "custom-counter-style",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("url('sqpurple.gif') custom-counter-style outside"),
+    [{
+      sourceIndex: 20,
+      type: "word",
+      value: "custom-counter-style",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("custom-counter-style url('sqpurple.gif') outside"),
+    [{
+      sourceIndex: 0,
+      type: "word",
+      value: "custom-counter-style",
+    }]
+  )
+  t.deepEqual(
+    findListStyleType("url('sqpurple.gif') outside custom-counter-style"),
+    [{
+      sourceIndex: 28,
+      type: "word",
```

Comment:
Oh I see, this is the object directly from postcss-value-parser. I think we should add a level of abstraction and make sure it's an object whose structure *we* define, so that if we switch value-parsers the tests still run.